components/text_input/README.md

```diff
@@ -0,0 +1,81 @@
+---
+description: Text inputs allow users to enter any combination of letters, numbers, or symbols of their choosing in a structured format.
+title: Text Input
+status: Draft
+---
```

Comment:
add author attributes (should be your github username)

Reply:
ok will do now